internal/hcs/process.go

```diff
@@ -411,7 +439,7 @@ func (process *Process) CloseStdout(ctx context.Context) (err error) {
 	defer process.handleLock.Unlock()
 
 	if process.handle == 0 {
-		return nil
+		return makeProcessError(process, operation, ErrAlreadyClosed, nil)
```

Comment:
What are the ramifications for this, we used to just noop, what scenario would this potentially change behavior (for better or for worse)

Reply:
So the only callers of `CloseStd*` is `internal\cmd.Cmd`, which will ignore.
The only motivation was standardizing with `gcs.Process`, which originally did return the error in its `CloseStd*` functions.

Reply:
Gotcha